agent-dynamic-configuration/AgentDynamicConfiguration.proto

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "org.apache.skywalking.apm.network.agent.dynamic.configuration.v3";
+option csharp_namespace = "SkyWalking.NetworkProtocol";
+option go_package = "skywalking/network/agent/configuration/v3";
+
+import "common/Common.proto";
+
+// Fetch the latest dynamic configurations of the service.
+service ConfigurationDiscoveryService {
+    rpc fetchConfigurations (ConfigurationSyncRequest) returns (Commands) {
```

Comment:
Still miss the documentation about the Command explanation. Please bring the conclusion of the discussions on the issue

Reply:
done, please help review again.

Reply:
I helped you on polishing the official documentation. Hope this will help and give you a good example about how to write the protocol comments.

Reply:
The key points of the protocol document is describing the logic rather than the codes like you did. We(SkyWalking) even can't predict how many languages/SDKs/agents will support this, when you put all words in Java, it would be super confused to other language developers. 
Java is the language we choose for backend, nothing more, and nothing special.

Reply:
I get it, thanks.